src/pperm.c

```diff
@@ -6322,9 +6212,18 @@ Obj OnSetsPPerm(Obj set, Obj f)
                     *ptres++ = tmp;
                 }
             }
-            else { /* this case cannot occur since I think POW is not defined
-                      */
-                ErrorQuit("not yet implemented!", 0L, 0L);
+            else {
+                // This case does not work since PowIntPPerm2/4 only work for
+                // INTOBJ's (i.e. small integers). The analogous code in
+                // permutat.c uses the macro POW, which calls PowIntPerm2/4,
+                // which if called with a non-small positive integer returns
+                // that integer, since every permutation fixes every positive
+                // integer (small or not). The methods for PowIntPPerm2/4 give
+                // an error if the argument is a non-small integer, and so we
+                // cannot call POW here.
```

Comment:
Actually, why does it do that? Why not also have a method for (positive) bigint+pperm which returns the bigint unchanged, mirroring the permutation code?

Reply:
Yes, we could do that, but for now this is not implemented, and so I opted to fix the comment and make the error message more meaningful.

Reply:
Also just to point out that partial perms unlike permutations do not fix all points where they are not defined, they are simply undefined on those points, so `PowIntPPerm2/4` should return `0` in those cases.